cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -54,10 +54,9 @@ object ReporterCommand : CommandWithHelp() {
             val REPORTERS = Reporter.ALL.associateBy { it.reporterName.toUpperCase() }
         }
 
-        override fun convert(name: String): Reporter {
-            return REPORTERS[name.toUpperCase()]
+        override fun convert(name: String) =
```

Comment:
prefer explicit return type here

Reply:
That's probably an edge case as the lookup happens in a map called `REPORTERS`, so it's quite obvious, but I've added the `Reporter` return type nonetheless.